ansible/roles/ocp4-workload-authentication/tasks/workload.yml

```diff
@@ -1,48 +1,64 @@
 ---
-# Implement your Workload deployment tasks here
-
-- name: Check that ocp4_idm_install is defined
+- name: Set up ocp4_workload_authentication combined dictionary
+  set_fact:
+    ocp4_workload_authentication: >-
+      {{ ocp4_workload_authentication_defaults
+       | combine(ocp4_workload_authentication_input  | default( {} ),
+                 ocp4_workload_authentication_secret | default( {} ))
```

Comment:
Don't you want the combine to be recursive: `combine( A, B, recursive=True)` ?

> The filter also accepts an optional recursive=True parameter to not only override keys in the first hash, but also recurse into nested hashes and merge their keys too

Reply:
Good point, to me this should be the default behavior but isn't. I agree `recursive=True`

Reply:
Good point. Done.